code/datums/quirks/neutral_quirks/transhumanist.dm

```diff
@@ -114,40 +114,45 @@
 	var/obj/item/new_part = new part_type()
 	if(isbodypart(new_part))
 		var/obj/item/bodypart/new_bodypart = new_part
-		slot_string = new_bodypart.plaintext_zone
+		slot_string = "[new_bodypart.plaintext_zone]"
 		old_part = human_holder.return_and_replace_bodypart(new_bodypart, special = TRUE)
 	else if(isorgan(new_part))
 		var/obj/item/organ/new_organ = new_part
 		old_part = human_holder.get_organ_slot(new_organ.slot)
 		if(new_organ.Insert(human_holder, special = TRUE))
 			old_part.moveToNullspace()
 			STOP_PROCESSING(SSobj, old_part)
-			slot_string = new_organ.name
+			slot_string = "[new_organ.name]"
```

Comment:
you reverted the changes where you got rid of the `"[]"` on accident

Reply:
oh oops